tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -218,7 +218,26 @@ def test_comparison_empty_comparison_period(self):
             "comparisonInterval": "1d",
         }
         rule = self.get_rule(data=data, rule=Rule(environment_id=None))
-        self.assertDoesNotPass(rule, event)
+        self.assertDoesNotPass(rule, event, is_new=False)
+
+    def test_is_new_issue_skips_snuba(self):
+        # Looking for more than 1 event
+        data = {"interval": "1m", "value": 6}
+        minutes = 1
+        rule = self.get_rule(data=data, rule=Rule(environment_id=None))
+        environment_rule = self.get_rule(data=data, rule=Rule(environment_id=self.environment.id))
+
+        event = self.add_event(
+            data={
+                "fingerprint": ["something_random"],
+                "user": {"id": uuid4().hex},
+            },
+            project_id=self.project.id,
+            timestamp=before_now(minutes=minutes),
+        )
+        # Issue is new and is the first event
+        self.assertDoesNotPass(rule, event, is_new=True)
+        self.assertDoesNotPass(environment_rule, event, is_new=True)
 
```

Comment:
This doesn't really test what it says, since this would also pass without your changes, since the count would be < 6. Worth mocking `get_rate` and checking that it doesn't get called?